### PR TITLE
[uio50] scrolls

### PR DESCRIPTION
**Pull request by SofiMat**

### CODE REVIEW

File: client/src/app/common/Header/HeaderMain.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```
Comment (Hollister009) [inline client/src/app/common/Header/HeaderMain.js]: I think, that I have already fixed this. So remove these lines

File: client/src/app/shared/ProductItem/ExtendedDetails.js [hunk 2]
```diff
@@ -117,8 +120,13 @@ class ViewDetailsFull extends Component {
             />
             <h4 className="highlighted">{title}</h4>
           </Link>
-          <div className={styles.full__sizes}>{`sizes : ${allSizes}`}</div>
-          <div className={styles.full__swatches}>{swatches}</div>
+          <div className={styles.full__sizes}>
+            <div className={styles.sizesText}>Sizes: </div>
```
Comment (Hollister009) [inline client/src/app/shared/ProductItem/ExtendedDetails.js]: So many `<div>`, why not use more semantic elements (`<p>`, `<h3>`)?

File: client/src/app/pages/page404.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ class Page404 extends Component {
           </div>
           <div>
             <img
-              src="https://res.cloudinary.com/so/image/upload/v1551445566/logos/oie_11438wBPVT3CE.gif"
+              src="https://res.cloudinary.com/sofimat/image/upload/v1551793877/empty/oie_11438wBPVT3CE.gif"
```
Comment (Hollister009) [inline client/src/app/pages/page404.js]: Next time, do not be so lazy and import it from the **project** directory.

File: client/src/app/components/Cart/ItemInCart.js [hunk 4]
```diff
@@ -131,8 +131,12 @@ class ItemInCart extends Component {
         <div className={styles.cart_block}>
           <img src={item.src} alt={item.title} className={styles.item_image} />
           <div className={styles.item_title}>{item.title}</div>
-          <div className={styles.select_colors}>{swatches}</div>
-          <div className={styles.select_sizes}>{sizes}</div>
+          <div className={styles.select_colors}>
+            <div className={styles.scrollColor}>{swatches}</div>
```
Comment (Hollister009) [inline client/src/app/components/Cart/ItemInCart.js]: Where is semantic ?

File: client/src/app/components/Cart/Cart.js [hunk 5]
```diff
@@ -56,14 +56,14 @@ class Cart extends Component {
           <div className={styles.empty_cart}>
             <Link to="/products">
               <img
-                src="https://res.cloudinary.com/so/image/upload/v1551445634/logos/cart.gif"
+                src="https://res.cloudinary.com/sofimat/image/upload/v1551793878/empty/cart.gif"
```
Comment (Hollister009) [inline client/src/app/components/Cart/Cart.js]: I don't like such lazy aproach.
What are u going to do if this server will expire?

File: client/src/app/common/Header/HeaderMain.js (hunk 1 above)
Comment (SofiMat) [reply client/src/app/common/Header/HeaderMain.js]: +

File: client/src/app/pages/page404.js (hunk 3 above)
Comment (SofiMat) [reply client/src/app/pages/page404.js]: sure)

File: client/src/app/components/Cart/Cart.js (hunk 5 above)
Comment (SofiMat) [reply client/src/app/components/Cart/Cart.js]: well, it's quite reliable resource, and our database is built on it

File: client/src/app/components/Cart/ItemInCart.js (hunk 4 above)
Comment (SofiMat) [reply client/src/app/components/Cart/ItemInCart.js]: I have no idea where to apply it to

File: client/src/app/shared/ProductItem/ExtendedDetails.js (hunk 2 above)
Comment (SofiMat) [reply client/src/app/shared/ProductItem/ExtendedDetails.js]: ok, i'll change

File: client/src/app/components/Cart/Cart.js (hunk 5 above)
Comment (SofiMat) [reply client/src/app/components/Cart/Cart.js]: done

File: client/src/app/shared/ProductItem/ExtendedDetails.js (hunk 2 above)
Comment (SofiMat) [reply client/src/app/shared/ProductItem/ExtendedDetails.js]: done

File: client/src/app/common/Header/HeaderMain.js (hunk 1 above)
Comment (SofiMat) [reply client/src/app/common/Header/HeaderMain.js]: done

**Pull request merged**